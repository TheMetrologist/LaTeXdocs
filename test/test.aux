\relax 
\@writefile{toc}{\contentsline {section}{\numberline {0.1}Test Section}{6}}
\@writefile{toc}{\contentsline {subsection}{\numberline {0.1.1}Subsection}{6}}
\newlabel{foot1}{{1}{6}}
\@writefile{lof}{\contentsline {figure}{\numberline {1}{\ignorespaces LEMAS system\relax }}{7}}
\providecommand*\caption@xref[2]{\@setref\relax\@undefined{#1}}
\newlabel{fig:LEMAS}{{1}{7}}
\newlabel{fig:2}{{2b}{7}}
\newlabel{sub@fig:2}{{b}{7}}
\@writefile{lof}{\contentsline {figure}{\numberline {2}{\ignorespaces Who approves?\relax }}{7}}
