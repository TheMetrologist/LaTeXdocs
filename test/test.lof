\contentsline {figure}{\numberline {1}{\ignorespaces LEMAS system\relax }}{7}
\contentsline {figure}{\numberline {2}{\ignorespaces Who approves?\relax }}{7}
