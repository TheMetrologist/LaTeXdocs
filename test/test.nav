\beamer@endinputifotherversion {3.24pt}
\headcommand {\slideentry {0}{0}{1}{1/1}{}{0}}
\headcommand {\beamer@framepages {1}{1}}
\headcommand {\sectionentry {1}{Test Section}{3}{Test Section}{0}}
\headcommand {\beamer@sectionpages {1}{2}}
\headcommand {\beamer@subsectionpages {1}{2}}
\headcommand {\beamer@subsectionentry {0}{1}{1}{3}{Subsection}}\headcommand {\beamer@subsectionpages {3}{2}}
\headcommand {\beamer@partpages {1}{5}}
\headcommand {\beamer@subsectionpages {3}{5}}
\headcommand {\beamer@sectionpages {3}{5}}
\headcommand {\beamer@documentpages {5}}
\headcommand {\def \inserttotalframenumber {1}}
