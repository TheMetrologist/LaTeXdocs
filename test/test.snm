\beamer@slide {fig:LEMAS}{4}
\beamer@slide {fig:2}{5}
\beamer@slide {sub@fig:2}{5}
