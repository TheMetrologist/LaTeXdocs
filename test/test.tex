\documentclass{proc}
% \documentclass{article}
% \documentclass{book}
\title{Latex Test}
\author{Michael Braine}
\usepackage{amsmath}
\usepackage{graphicx}
\usepackage{subcaption}

\begin{document}
  \maketitle
  \tableofcontents
  \listoffigures
  \newpage
  Test text knows no bounds!
  \section{Test Section}
  Where my homies at
  \subsection{Subsection}
  Structuring
  \begin{equation}
    f(x) = e^x + g(x)
  \end{equation}

  \begin{align*}
    f(x) &= e^x + g(x)\\
    f(x) - g(x) &= e^x
  \end{align*}

  \begin{align*}
    f(x) &= x^2\\
    g(x) &= \frac{1}{x}\\
    F(x) &= \int^a_b \frac{1}{3}x^3\\
    \frac{1}{\sqrt{x}}
  \end{align*}

  $\left[
    \begin{matrix}
      one & zero\\
      0 & 1
    \end{matrix}
  \right]$

  \begin{equation}
    \left[
      \begin{matrix}
        one & zero\\
        0 & 1
      \end{matrix}
    \right]
  \end{equation}

  $\left(\frac{1}{\sqrt{x}}\right)$

  \begin{figure}
    \includegraphics[width=\linewidth]{figures/index.jpeg}
    \caption{LEMAS system}
    \label{fig:LEMAS}
  \end{figure}

  As we can see in Figure \ref{fig:LEMAS}, this is the system.

  \begin{figure}
    \centering
    \begin{subfigure}{0.3\linewidth}
      \includegraphics[width=\linewidth]{figures/fallout.jpeg}
      \caption{Mr. Fallout boy approves}
    \end{subfigure}
    \begin{subfigure}{0.5\linewidth}
      \includegraphics[width=\linewidth]{figures/index.jpeg}
      \caption{LEMAS system}
      \label{fig:2}
    \end{subfigure}
    \caption{Who approves?}
  \end{figure}

  The world doesn't revolve around itself\footnote{\label{foot1}It really doesn't}

\end{document}
