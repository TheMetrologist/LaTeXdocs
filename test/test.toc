\contentsline {section}{\numberline {0.1}Test Section}{6}
\contentsline {subsection}{\numberline {0.1.1}Subsection}{6}
