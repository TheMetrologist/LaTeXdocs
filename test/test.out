\BOOKMARK [2][]{Outline0.1}{Test Section}{}% 1
\BOOKMARK [3][]{Outline0.1.1.3}{Subsection}{Outline0.1}% 2
